### PR TITLE
Admin system management interface

### PR DESCRIPTION
**Pull request by haonh**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,17 @@
 Rails.application.routes.draw do
+
+  namespace :admin do
+  get 'words/index'
+  end
```
Comment (nhs3108) [inline config/routes.rb]: Delete

File: app/controllers/admin/dashboard_controller.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Admin::DashboardController < ApplicationController
+  def index
+  	@users_length = User.count
```
Comment (ttuan) [inline app/controllers/admin/dashboard_controller.rb]: ident

File: app/views/admin/dashboard/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<%= provide :title, t("dashboard").titleize %>
+<h1><%= t "admin_header" %></h1>
+<div class="row">
+  <div class="col-md-3">
+    <a href="<%= admin_users_path %>">
+      <span class="glyphicon glyphicon-user" style="font-size: 80px"><%= @users_length %> </span>
```
Comment (nhs3108) [inline app/views/admin/dashboard/index.html.erb]: Write style in css file

File: app/assets/stylesheets/custom.css.scss [hunk 4]
```diff
@@ -117,6 +117,7 @@ checkbox {
   margin-left: 0;
 }
```
Comment (thiennv57) [inline app/assets/stylesheets/custom.css.scss]: remove this line

File: app/views/admin/dashboard/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    </a>
+  </div>
+  <div class="col-md-3">
+    <a href="<%= admin_categories_path %>">
+      <span class="glyphicon glyphicon-book" style="font-size: 80px"><%= @categories_length %></span>
```
Comment (nhs3108) [inline app/views/admin/dashboard/index.html.erb]: Write style in css file

File: app/controllers/admin/users_controller.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+class Admin::UsersController < ApplicationController
+  def index
+  	users_length = User.count
```
Comment (ttuan) [inline app/controllers/admin/users_controller.rb]: ident

File: app/views/admin/dashboard/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    </a>
+  </div>
+  <div class="col-md-3">
+    <a href="<%= admin_words_path %>">
+      <span class="glyphicon glyphicon-font" style="font-size: 80px"><%= @words_length %></span>
```
Comment (nhs3108) [inline app/views/admin/dashboard/index.html.erb]: Write style in css file

File: app/views/admin/dashboard/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+    </a>
+  </div>
+  <div class="col-md-3">
+    <a href="<%= admin_lessons_path %>">
+      <span class="glyphicon glyphicon-file" style="font-size: 80px"><%= @lessons_length %></span>
```
Comment (nhs3108) [inline app/views/admin/dashboard/index.html.erb]: Write style in css file

File: app/views/admin/dashboard/index.html.erb (hunk 3 above)
Comment (ttuan) [inline app/views/admin/dashboard/index.html.erb]: Move style to css file

File: app/views/admin/dashboard/index.html.erb (hunk 5 above)
Comment (ttuan) [inline app/views/admin/dashboard/index.html.erb]: move style to css

File: app/views/admin/dashboard/index.html.erb [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+      <span class="glyphicon glyphicon-book" style="font-size: 80px"><%= @categories_length %></span>
+    </a>
+  </div>
+  <div class="col-md-3">
+    <a href="<%= admin_words_path %>">
+      <span class="glyphicon glyphicon-font" style="font-size: 80px"><%= @words_length %></span>
+    </a>
+  </div>
+  <div class="col-md-3">
+    <a href="<%= admin_lessons_path %>">
```
Comment (thiennv57) [inline app/views/admin/dashboard/index.html.erb]: link_to

File: app/views/admin/dashboard/index.html.erb [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+      <span class="glyphicon glyphicon-user" style="font-size: 80px"><%= @users_length %> </span>
+    </a>
+  </div>
+  <div class="col-md-3">
+    <a href="<%= admin_categories_path %>">
+      <span class="glyphicon glyphicon-book" style="font-size: 80px"><%= @categories_length %></span>
+    </a>
+  </div>
+  <div class="col-md-3">
+    <a href="<%= admin_words_path %>">
```
Comment (thiennv57) [inline app/views/admin/dashboard/index.html.erb]: link_to

File: config/routes.rb [hunk 11]
```diff
@@ -1,4 +1,17 @@
 Rails.application.routes.draw do
+
```
Comment (ttuan) [inline config/routes.rb]: delete this line

File: config/routes.rb [hunk 12]
```diff
@@ -1,4 +1,17 @@
...
+  namespace :admin do
+  get 'words/index'
+  end
+
+  namespace :admin do
+   root "dashboard#index"
```
Comment (ttuan) [inline config/routes.rb]: ident

File: app/assets/stylesheets/custom.css.scss [hunk 13]
```diff
@@ -118,6 +118,7 @@ checkbox {
...
 
 input, textarea, select, .uneditable-input {
+
```
Comment (ThuBM) [inline app/assets/stylesheets/custom.css.scss]: remove this line

**Comment by ThuBM** [comment]
OK

**Pull request merged**